applications/luci-app-upnp/htdocs/luci-static/resources/view/status/include/80_upnp.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+'require rpc';
+'require uci';
+
+var callUpnpDeleteRule;
+
+callUpnpDeleteRule = rpc.declare({
+	object: 'luci.upnp',
+	method: 'delete_rule',
+	params: [ 'token' ],
+	expect: { result : "OK" },
+});
+
+return L.Class.extend({
+	title: _('Active UPnP Redirects'),
+
+	callUpnpGetStatus: rpc.declare({
```

Comment:
Can you move this function to the top as well and declare it with `var` ? It's no functional change but will make it more consistent.